### ISSUE TITLE
evil-mc does not working perfect with paredit-mode

### ISSUE THREAD

**Issue by 0x4ec7** [gabesoft/evil-mc#32]
Hi Gabriel,

I am using paredit-mode and evil-mc-mode at the same time, when working with multiple cursors in insert-state, __backspace__ does not working as expected.

evil-mc-mode_
evil-mc-mode_
evil-mc-mode_|

__backspace__:

evil-mc-mode_
evil-mc-mode_
evil-mc-mod_|

I found that __backspace__ is binded to __paredit-backward-delete__. Is there a way to fix this?  

Thanks!

**Comment by gabesoft** [comment]
Yes, we  need to add the paredit commands. Similar to how it's done here https://github.com/gabesoft/evil-mc/pull/17 for evil-cleverparens. The commands have moved to this [https://github.com/gabesoft/evil-mc/blob/master/evil-mc-known-commands.el](file) though.

**Comment by 0x4ec7** [comment]
It turns out that I have another parenthesis package(smartparens) involved in my configuration. so I  removed paredit, now it works. 

In the mean time, I found that expand-region doesn't working well with emc. expand-region is a very useful package, It would be great if this package is supported. (I have a work-around now).
